web/docs/how-does-tenzir-work/README.md

```diff
@@ -0,0 +1,66 @@
+# How Does Tenzir Work?
+
+Tenzir revolves around three primary components:
+
+- **Pipeline**: A defined sequence of operators responsible for loading,
+  parsing, transforming, and routing data. Pipelines are the core mechanism for
+  data processing.
+- **Node**: A running process that manages and executes pipelines.
+- **Platform**: A higher-level management layer that provides oversight and
+  control over multiple nodes.
+
+## Standalone and Managed Pipelines
+
+Tenzir offers flexibility in running pipelines through two main modes:
+
+1. **Standalone Execution**: Run directly from the command line using the
+   `tenzir` binary. This mode is ideal for quick, manual data transformations
+   akin to how one would use `jq`, but with broader data handling capabilities.
+2. **Managed Execution via Nodes**: For more sustained or automated data
+   processing, pipelines can be managed within a node. This is either launched
+   as a Docker container or by running the `tenzir-node` binary.
```

Comment:
Not clear to me how what "this" refers to. I assume "the node", but the sentence is not structured that way.